src/styles/global.css

```
@@ -201,19 +198,9 @@
-  ascent-override: 74%;
-  descent-override: normal;
-  line-gap-override: normal;
-}
-
-@font-face {
-  font-family: 'Cinzel';
-  src: url('/fonts/cinzel-regular.woff2') format('woff2');
-  font-weight: 400;
-  font-display: swap;
+  font-family: 'Cinzel Fallback';
+  src: local(Georgia);
```

Stylelint (17.12.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)